<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookManagement</class>
 <widget class="QWidget" name="BookManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>791</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="add_book">
    <attribute name="title">
     <string>Add New Book</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>381</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="book_title_lbl">
        <property name="text">
         <string>Book Title: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="title_lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="author_lbl">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="author_lineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="genre_lbl">
        <property name="text">
         <string>Genre: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="genre_comboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="pubyear_lbl">
        <property name="text">
         <string>Year Published: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="pubyear_spinBox">
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="rating_lbl">
        <property name="text">
         <string>GoodReads Rating: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="rating_doubleSpinBox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="desc_lbl">
        <property name="text">
         <string>Description (optional): </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="desc_lineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="add_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>360</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Book To Collection</string>
     </property>
    </widget>
    <widget class="QLabel" name="add_msg_lbl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>410</y>
       <width>461</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="remove_book">
    <attribute name="title">
     <string>Remove a Book</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>551</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="title_remove_lbl">
        <property name="text">
         <string>Book Title: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="title_remove_lineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="remove_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Book From Collection</string>
     </property>
    </widget>
    <widget class="QLabel" name="remove_msg_lbl">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>491</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
